Makefile

```diff
@@ -418,7 +418,8 @@ startup_files: include_dirs
 	#
 	@mkdir -p "$(OBJDIR)"
 	cd "$(OBJDIR)" && \
-	"$(WASM_CC)" $(WASM_CFLAGS) -c $(CRT_SOURCES) -MD -MP && \
+	"$(WASM_CC)" $(WASM_CFLAGS) -c $(CRT1_SOURCE) -MD -MP && \
```

Comment:
Can you add `-o crt1.o` to this line for clarity?

Also, can you avoid the "cd" and put the OBJDIR in the `-o` flag to avoid the line continuation stuff?

Reply:
In this long run I think it will be nicer to move the common code here into libc.a and to have these two be separate source files (crt1.c + crt1-reactor.c) that contain very little code.

Reply:
`-o crt1.o` and the line continuation stuff is now cleaned up.

I've also started refactoring this code, but there's more to do.